src/experimentConfig.ts

```diff
@@ -54,24 +62,18 @@ const generateExperimentConfigVal = <T>(key: keyof typeof experimentDistribution
 // values are the variant values that differ from the default feature
 // behavior/appearance, while the last value represents unchanged
 // behavior/appearance.
-// If no generateCfgValFn given, returns defined defaults
-// Error: Return type annotation circularly references itself.
-const asExperimentConfig: (isDefault?: boolean) => Cleaner<ExperimentConfig> = isDefault =>
-  asObject({
-    swipeLastUsp: asOptional<'true' | 'false'>(asValue('true', 'false'), isDefault ? 'false' : generateExperimentConfigVal('swipeLastUsp', ['true', 'false'])),
-    createAccountType: asOptional<CreateAccountType>(
-      asValue('light', 'full'),
-      isDefault ? 'full' : generateExperimentConfigVal('createAccountType', ['light', 'full'])
-    ),
-    legacyLanding: asOptional<'legacyLanding' | 'uspLanding'>(
-      asValue('legacyLanding', 'uspLanding'),
-      isDefault ? 'uspLanding' : generateExperimentConfigVal('legacyLanding', ['legacyLanding', 'uspLanding'])
-    ),
-    createAccountText: asOptional<'signUp' | 'getStarted' | 'createAccount'>(
-      asValue('signUp', 'getStarted', 'createAccount'),
-      isDefault ? 'createAccount' : generateExperimentConfigVal('createAccountText', ['signUp', 'getStarted', 'createAccount'])
-    )
-  })
+const asExperimentConfig: Cleaner<ExperimentConfig> = asObject({
+  swipeLastUsp: asOptional<'true' | 'false'>(asValue('true', 'false'), generateExperimentConfigVal('swipeLastUsp', ['true', 'false'])),
```

Comment:
Since this is a "simplification" PR, simplify further by removing the <> type notation. It's not needed. ie just use

`swipeLastUsp: asOptional(asValue('true', 'false')...`